packages/babel-plugin-polyfill-corejs3/test/fixtures/usage-pure/built-in-instance-methods/output.js

```diff
@@ -128,7 +132,7 @@ _reduceRightInstanceProperty(object);
 
 _repeatInstanceProperty(object);
 
-object.replaceAll;
```

Comment:
Since the `replaceAll` is stage 4, we should polyfill it if users are using an older version of corejs which provides `esnext.string.replace-all`.